### DISCUSSION TITLE
Checkboxes should be positive / affirmative (check to enable), not negative (check to disable)

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13657] [Feature Requests]
Most of the checkboxes in Directus are for positive actions: 'Allow feature to happen' where a checked box means that feature is enabled / active, but there are some places where you check the box to disable a feature. This is weird. A check in a box is usually (always?) a positive action to enable something, and no check means the feature is disabled.

## Examples
Positive checkboxes for allowing things:
![Screenshot from 2022-06-01 08-55-45](https://user-images.githubusercontent.com/95102939/171356260-59793b3c-db1b-4cec-9b38-b6c9bb4dae59.png)

Two negative checkboxes for checking to disable and one positive:

![Screenshot from 2022-06-01 08-56-22](https://user-images.githubusercontent.com/95102939/171356355-40bad653-e6a4-47df-b040-8ea3fd5235bb.png)

Ok! So you can argue you're adding a 'hidden' attribute, but from the user's point of view, you're actually removing the visibility of the item

---
I think the 'Disable' should become a pre-checked 'Allow' like in the first example, and 'hidden' should become a pre-checked 'Visible'.

So, a quick mock-up:
![Screenshot from 2022-06-01 09-02-57](https://user-images.githubusercontent.com/95102939/171357187-3c5a223a-d362-479c-9852-89765eced39f.png)

I'm going to guess this will break a lot of things like translation keys and possibly backwards compatibility?

**Comment by benhaynes** [comment]
**Readonly:** If you look at the label ("readonly") and then see it selected... you'll think it's readonly. The alternative would be to make the option "Editable", but that is not the standard. So I think this makes the most sense as-is.

**Hidden:** This one _does_ have a positive action... but "hidden" is being interpreted as a "negative" option. An interesting concept. Is a positive action for a negative option... negative?? 🤔

**Comment by rijkvanzanten** [comment]
> Ok! So you can argue you're adding a 'hidden' attribute, but from the user's point of view, you're actually removing the visibility of the item

I don't know if I agree with that! To me, the visible state feels like the default, and the "I want to hide it" is the _positive_ action here I'm enabling

**Reply by sxsws** [reply]
I humbly disagree with you both! 

I think there's two things at play here - the database first approach to the admin/collection side with its own terminology, and the 'detail/index' side with its own terminology which is taking that database structure and making it accessible for the user via forms.

I think these options for the UI sits firmly in the middle!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
